FORMAT-GUIDE.md

```diff
@@ -60,9 +60,16 @@ You can also use this to target specific events, for instance:  \
     Youtube Videos can be embedded by doing:
      - Add an image in the document. The image can be a screenshot of the video for instance but it doesn't really matter since it won't be displayed but replaced by the embedded video.
      - Add an "Alt Text" to the image by doing **Cmd+Opt+Y** or **Right click > "Alt Text..."**
-     - Put a Youtube video link in the **Description** field of the Alt Text. in the format `https://www.yoututbe.com/watch?v=[video_ID]`
+     - Put a Youtube video link in the **Description** field of the Alt Text. in the format `https://www.youtube.com/watch?v=[video_ID]`
     > Specifying a start time is not supported at this time.
 
+1. Embedded Iframes
+
+    Iframes can be embedded by doing:
+     - Add an image in the document. The image can be a screenshot of the iframe for instance but it doesn't really matter since it won't be displayed but replaced by the embedded iframe.
+     - Add an "Alt Text" to the image by doing **Cmd+Opt+Y** or **Right click > "Alt Text..."**
+     - Put a full URL in the **Description** field of the Alt Text. in the format `https://www.domain.com/watch?foo=bar`
```

Comment:
Add a note that only httpS is supported and not http? :)

Reply:
good point, fixed.